cookbook/tools/linkup_tools.py

```diff
@@ -0,0 +1,5 @@
+from agno.agent import Agent
+from agno.tools.linkup import LinkupTools
+
+agent = Agent(tools=[LinkupTools()], show_tool_calls=True)
+agent.print_response("Whats happening in France?", markdown=True)
```

Comment:
Question could be improved ("Whats" -> "What's" and maybe find something a bit more specific, e.g. "What are the latest news in French politics" or something like that)